kmeta/names_test.go

```diff
@@ -56,7 +56,12 @@ func TestChildName(t *testing.T) {
 		parent: "aaaa",
 		suffix: strings.Repeat("b---a", 20),
 		want:   "aaaa7a3f7966594e3f0849720eced8212c18b---ab---ab---ab---ab---ab",
-	}}
+	}, {
+		parent: strings.Repeat("a", 17),
+		suffix: "a.-.-.-.-.-.-.aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+		want:   "aaaaaaaaaaaaaaaaaa1eb10dc911444f8434af83b7225442da",
```

Comment:
It looks like we have no tests with capital letters. I suspect that's because we don't want names with capital letters.

Can you add a test on 63 or so that checks `IsDNS1123Subdomain` on the strings as well as a double-check?

(This will help if we add fuzzing tests later.)

Reply:
ah okay, let me make sure I understand...

so in addition to checking that want == got, we add an extra check to make sure got is a InDNS1123Subdomain?